cmsplugin_articles_ai/templates/cmsplugin_articles_ai/tag.html

```diff
@@ -0,0 +1,5 @@
+<span class="label label-default">
+    <a href="{% url 'tagged_articles' tag=tag.name %}" style="color: white;">
```

Comment:
It's just a reference template for getting started. For a few style definitions there is no point for any heftier styles.